src/large_scale_condensation.jl

```diff
@@ -0,0 +1,47 @@
+function get_large_scale_condensation_tendencies!(
+    Diag::DiagnosticVariables{NF},
+    M::ModelSetup{NF},
+) where {NF<:AbstractFloat}
+    @unpack pres_ref, cp, alhc, alhs, gravity, τ, RH¹, ΔRH, rhb = M.constants  # Let's give some of these constants better names?
+    @unpack nlon, nlat, nlev, σ_levels_half, σ_levels_full, σ_levels_thick =
+        M.geospectral.geometry
+    @unpack temp_grid, humid_grid, pres_surf_grid, cloud_top, precipitation_ls = Diag.grid_variables
+    @unpack temp_tend, humid_tend = Diag.parametrization_tendencies
+    @unpack cloud_top = Diag.intermediate_variables
+
+    # Use mathematical notation for consistency with the documentation
+    Q = humid_grid
+    T = temp_grid
+    ∂Q = humid_tend
+    ∂T = temp_tend
+
+    p = exp.(pres_surf_grid)  # Normalised surface pressure - TODO(alistair): check pressure units throughout
+    Qsat = get_saturation_specific_humidity(T, p, M)
+
+    # 1. Tendencies of humidity and temperature due to large-scale condensation
+    @inbounds for k = 2:nlev
```

Comment:
any idea why this starts at 2? Just trying to decipher whether something is not applied in the stratosphere (k=1,2)

Reply:
The original [docs](http://users.ictp.it/~kucharsk/speedy_description/km_ver41_appendixA.pdf) apply this parameterisation in tropospheric layers, which would seem to explain it. But is k=2 also a stratospheric layer? Naive question but how exactly do we know...?

Reply:
fortran speedy uses the top two levels as the stratosphere. This is hardcoded (e.g. some loops just start at 3), I've tried to generalize that by introducing `n_stratosphere_levels` in `default_parameters.jl` such that this parameter should be used instead given that we may want to have more levels with higher resolution

Reply:
Good to know. Hard to explain the loop starting at two then, but I guess they're just saying that some large-scale condensation occurs in the lower troposphere but not upper?

In any case good to be aware of this as we experiment with more vertical levels.

Reply:
Yes, I honestly believe there to many of theses lines in the Fortran code where it's absolutely unclear why certain choices have been made. I'd suggest to go with your best guess how this generalizes but add a comment like `# used to be k = 2:nlev` so that if we realize that something is wrong we know where we attempted to remove some hardcoding

Reply:
I like it :+1: